### DISCUSSION TITLE
couldn't get resource list for metrics.k8s.io/v1beta1: the server is currently unable to handle the request

### DISCUSSION THREAD

**Discussion by gauriballal** [k3s-io/k3s#7227] [Q&A]
After installing v1.26.3+k3s1 on open suse 15.4leap, facing 'couldn't get resource list for metrics.k8s.io/v1beta1: the server is currently unable to handle the request' error. Tried to download the latest component.yaml from metrics latest release but facing same issue. Everything worked fine with k3s v1.25.7.

**Comment by brandond** [comment]
Have you checked to see why the pod isn't running or can't scrape metrics? `kubectl get pod -A`, `kubectl logs` from the metrics-server pod, all would be good places to start, before going and trying to install stuff from other sources...

**Comment by Aysnine** [comment]
Be careful your proxy. Don't missing `NO_PROXY` for k3s. https://docs.k3s.io/advanced#configuring-an-http-proxy

**Comment by SpringHgui** [comment]
i solved by edit  `/etc/resolv.conf`
